### ISSUE TITLE
Presenting STPopup for modal controller (iPad)

### ISSUE THREAD

**Issue by viking2009** [kevin0571/STPopup#34]
Hi.
When I tried to present STPopupController from view controller with modal presentation style UIModalPresentationFormSheet, I got wrong frames for STPopup content controller.

![simulator screen shot 22 2015 16 13 34](https://cloud.githubusercontent.com/assets/398568/11957070/bce9b0ce-a8c7-11e5-8174-c2a40bc4d47a.png)

I think the problem is here:
```objc
- (void)animateTransition:(id <UIViewControllerContextTransitioning>)transitionContext
{
    UIViewController *fromViewController = [transitionContext viewControllerForKey:UITransitionContextFromViewControllerKey];
    UIViewController *toViewController = [transitionContext viewControllerForKey:UITransitionContextToViewControllerKey];
    
    toViewController.view.frame = fromViewController.view.frame;
```
Why are you changing toViewController.view.frame?

Thanks

**Comment by kmuddapu** [comment]
Hello ,

I am trying to change the frame, but not able to fix it !

Can any one help me !!

Thank You,

**Comment by nottix** [comment]
+1

**Comment by kevin0571** [comment]
Fixed in 1.7.1
